### PR TITLE
Men 4518: Integration test

### PR DESCRIPTION
**Pull request by oleorhagen**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -31,3 +31,5 @@ docker-compose.override.yml
 backend-tests/acceptance.*
 backend-tests/report_backend_integration_enterprise.html
 backend-tests/results_backend_integration_enterprise.xml
+
+.gitattributes
```
Comment (kacf) [inline .gitignore:35]: Why? This is a very common file to have in a repository. There is a separate place for "your own" gitattributes, they should be put in `$GIT_DIR/info/attributes`.

**Review by kacf — APPROVED** [review]
I would omit the gitignore entry (see comment), but otherwise looks beautiful!

**Comment by oleorhagen** [comment]
Pipe passed last time around, only difference is the `.gitignore` commit has been removed. 

Merging

**Comment by lluiscampos** [comment]
:+1:

**Pull request merged**